tutorials/ghost-on-app-engine-part-1-deploying.md

```diff
@@ -293,9 +281,41 @@ Create a `config.js` file from the default config file:
 
     Read more about [using `app.yaml`][appyaml].
 
+1. Migrate the database to allow use in production, with:
+
+        NODE_ENV=production knex-migrator init --mgpath node_modules/ghost
+
+1. Add `"socketPath": "/cloudsql/YOUR_INSTANCE_NAME"` in the connection properties section of your `config.production.json`, so you end up with:
+
+        {
+            "url": "http://YOUR_PROJECT_ID.appspot.com",
+            "fileStorage": false,
+            "mail": {},
+            "database": {
+                "client": "mysql",
+                "connection": {
+                    "socketPath": "/cloudsql/YOUR_INSTANCE_NAME",
+                    "user": YOUR_MYSQL_USERNAME,
+                    "password": YOUR_MYSQL_PASSWORD,
+                    "database": YOUR_MYSQL_DATABASE_NAME,
+                    "charset": "utf8"
+                },
+                "debug": false
+            },
+            "server": {
+                "host": "0.0.0.0",
+                "port": "8080"
+            },
+            "paths": {
+                "contentPath": "content/"
+            }
+        }
+
+It's very important that you only do this step after migrating the database, because the socketPath property is required to deploy on Google App Engine, but it causes knex-migrator to throw an error.
```

Comment:
I would split this into 2 sentences:
It's very important that you only do this step after migrating the database. The *socketPath* property is required to deploy on Google App Engine, but it causes *knex-migrator* to throw an error.